database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Build/85_DML_PROPERTY_LEASE.sql

```diff
@@ -0,0 +1,441 @@
+INSERT INTO PIMS_PROPERTY_LEASE (PROPERTY_ID, LEASE_ID)
```

Comment:
This script fails as the following columns are missing: [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY]

Reply:
Works for me.  They have default values that should work for you but I can add them to the script.

Reply:
No defaults in my version of the database, perhaps that got missed?

Reply:
Seems like the defaults are potentially an issue with the migrations. More investigation required, logging an issue.